engine-rest/engine-rest-openapi/src/main/templates/main.ftl

```diff
@@ -32,7 +32,8 @@
   ],
   "tags": [
     {"name": "Process instance"},
-    {"name": "Deployment"}
```

Comment:
let's adjust this to alphabetical order:
```
{"name": "Deployment"},
{"name": "Engine"},
{"name": "Process instance"}
```

Reply:
I'll add this one with the PR #715 since I'm currently working on it.